lua/MethHelper.lua

```diff
@@ -103,8 +106,64 @@ currentRecipeList['Muriatic Acid'] = false
 currentRecipeList['Caustic Soda'] = false
 currentRecipeList['Hydrogen Chloride'] = false
 
-IngredientCount = 0
-OldIngredientCount = 0
+
+local lastAdded = false
+local IngredientCount = 0
+local OldIngredientCount = 0
+
+function sendMethMessage(number, id)
```

Comment:
Can we clean this up slightly so it has less repetition?

```lua
    MessageColor = Color.green;

    if number == 1 then
        Message =  'Ingredient added!';
    elseif number == 2 then
        Message = '[' .. countAddedIngredients(currentRecipeList) .. '/3] [' .. ingredient_dialog[id] .. ']';    
    elseif number == 3 then
        Message = 'Total bags: [' .. totalBags .. ']';
    elseif number == 4 then
        Message = 'No finished dialouge caught, Bag-count is being guessed by now';
        MessageColor = Color.red;
    else
        return;
    end

    if MethHelper._data.silent_toggle == true or MethHelper._data.silent_toggle == 'on' then
            managers.chat:_receive_message(1, '[SilentMethMagic]', Message, MessageColor);
    else
            managers.chat:send_message(1, '[SilentMethMagic]',Message, MessageColor);
    end
```
(I feel like thats easier to understand)